internal/github/lastRepo.go

```diff
@@ -0,0 +1,66 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/shurcooL/githubv4"
+)
+
+type LastRepo struct {
+	Name        string
+	Description string
+	CreatedAt   time.Time
+	UpdatedAt   time.Time
+}
+
+func (c LastRepo) String() string {
+	msg := fmt.Sprintf("\nThe last repo updated \n\n **%v** \n \t %v \n Created At: %s \n Updated At: %s", c.Name, c.Description, c.CreatedAt.Format("2006-01-02"), c.UpdatedAt.Format("2006-01-02"))
+	if len(c.Name) == 0 {
+		msg = fmt.Sprintf("Could not find a repository")
+	}
+
+	return msg
+}
+
+func (g *GithubService) GetLastRepoByUsername(ctx context.Context, username string) (*LastRepo, error) {
```

Comment:
thoughts on adding Updated as qualifier in the function naming
`GetLastUpdatedRepoByUsername`?

Reply:
This also could just be an unnecessary nitpick.. if we need to make that distinction in the future,  we can

Reply:
Not a nitpick. better to be more descriptive with naming. I agree with this change completely.

Reply:
But now i have a question. the command the end user would use on discord is "!lastrepo". Should this be updated to "!lastupdatedrepo"?  

Keeping the end users commands short has ease of use advantages but if there is a command we may want to add later that is better suited to be called with "!lastrepo" we may want to change this. 

Thoughts?

Reply:
ah yeah good catch.. interesting
the command description is describing it pretty accurately but i would think the command should also show intent as close as possible
we can see how the more descriptive option feels